src/Hl7.Fhir.Core/Serialization/JsonSerializationOptionsExtensions.cs

```diff
@@ -25,26 +25,26 @@ public static class JsonSerializerOptionsExtensions
         /// <summary>
         /// Initialize the options to serialize using the JsonFhirConverter, producing compact output without whitespace.
         /// </summary>
-        public static JsonSerializerOptions ForFhir(this JsonSerializerOptions options) => options.ForFhir(typeof(ModelInfo).Assembly);
+        public static JsonSerializerOptions ForFhir(this JsonSerializerOptions options) => options.ForFhir(typeof(ModelInfoNEW).Assembly);
```

Comment:
I think these methods will have to become shims.

Reply:
Moved to the satellites projects as a shim.